tool/tsh/tsh.go

```diff
@@ -885,7 +887,9 @@ func Run(ctx context.Context, args []string, opts ...cliOption) error {
 	reqSearch.Flag("search", searchHelp).StringVar(&cf.SearchKeywords)
 	reqSearch.Flag("query", queryHelp).StringVar(&cf.PredicateExpression)
 	reqSearch.Flag("labels", labelHelp).StringVar(&cf.UserHost)
-
+	reqSearch.Flag("kube-cluster", "Kubernetes Cluster to search for Pods").StringVar(&cf.KubernetesCluster)
+	reqSearch.Flag("kube-namespace", "Kubernetes Namespace to search for Pods").Default(corev1.NamespaceDefault).StringVar(&cf.kubeNamespace)
+	reqSearch.Flag("all-namespaces", "Search Pods in every namespace").BoolVar(&cf.kubeAllNamespaces)
```

Comment:
For me, naming is inconsistent. On the one hand, you have `--kube-namespace` that suggest K8S namespace, but then `--all-namespaces` sounds confusing as the name doesn't says anything about which namespaces you are referring to.